[New Sample] Browse building floors (#673)
Esri/arcgis-runtime-samples-java: map/browse-building-floors/README.md

# Browse building floors

Display and browse through building floors from a floor-aware web map.

![Image of browse building floors](BrowseBuildingFloors.png)

## Use case

Having map data to aid indoor navigation in buildings with multiple floors such as airports, museums, or offices can be incredibly useful. For example, you may wish to browse through all available floor maps for an office in order to find the location of an upcoming meeting in advance.

## How to use the sample

> Use the spinner to browse different floor levels in the facility. Only the selected floor will be displayed.

Reviewer: ```suggestion
Use the dropdown menu to browse different floor levels in the facility. Only the selected floor will be displayed.
```

## How it works

1. Create a `PortalItem` using the identifier of a floor-aware web map.
2. Create a map using the portal item.
3. Create a map view and assign the map to it.
4. Wait for the map to load and retrieve the map's `FloorManager`.

> 5. Wait for the floor manager to load and get its list of  `FloorLevel`s.

Reviewer: ```suggestion
5. Wait for the floor manager to load and get its list of `FloorLevel`s.
```

> 6. Set all floor levels `isVisible` property to false.

Reviewer: ```suggestion
6. Set the `isVisible` property to false for all floor levels.
```
Can't decide which reads better. I've probably read the sentence too many times now 😄
